datarepo-clienttests/src/main/java/runner/DisruptionResult.java

```diff
@@ -0,0 +1,23 @@
+package runner;
+
+import edu.umd.cs.findbugs.annotations.SuppressFBWarnings;
+
+@SuppressFBWarnings(
+    value = "URF_UNREAD_PUBLIC_OR_PROTECTED_FIELD",
+    justification = "This POJO class is used for easy serialization to JSON using Jackson.")
+public class DisruptionResult {
```

Comment:
What is this result class used for?

Reply:
You're right, it's not needed! It's been removed.